index.html

```diff
@@ -181,7 +181,7 @@ <h1 class="card-title">About <br>Myself</h1>
                 <h2>LET’S CONNECT</h2>
```

Comment:
- [ ] On line 177, use a `h2` tag for the about section. this maintains heirachy for your headings. It is also a better practice to have not more that two `h1` tags per page